internal/command/project_application_oidc.go

```diff
@@ -359,3 +359,11 @@ func getOIDCAppWriteModel(ctx context.Context, filter preparation.FilterToQueryR
 	err = appWriteModel.Reduce()
 	return appWriteModel, err
 }
+
+func trimStringSliceWhiteSpaces(rawSlice []string) []string {
+	trimmedSlice := make([]string, 0)
+	for _, s := range rawSlice {
+		trimmedSlice = append(trimmedSlice, strings.TrimSpace(s))
+	}
+	return trimmedSlice
+}
```

Comment:
```suggestion
func trimStringSliceWhiteSpaces(slice []string) []string {
	for i, s := range slice {
		slice[i] = strings.TrimSpace(s)
	}
	return slice
}
```

Reply:
Totally @livio-a, cleaner code. I'll update the code ASAP. Thanks!!